### PR TITLE
Fixed about section

### PR DESCRIPTION
**Pull request by VC6378**
Fixed the About section, where the code was corrupted and also the images and content were being shown twice.

BEFORE:
image

AFTER:
image

### CODE REVIEW

**Comment by varun-singhh** [comment]
can you tell how to reproduce this bug I need to look it first.

**Comment by jainmudit68** [comment]
> can you tell how to reproduce this bug I need to look it first.

i have raised the issue #84 for the same yesterday itself i was waiting for it to be assigned and reviewed before creating a pull request

File: components/homepage/About/About.jsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const About = () => {
                     </div>
                 </div>
             </div>
-            <hr className="divider"></hr>
+            {/* <hr className="divider"></hr>
             <div className={style.feature}>
```
Comment (varun-singhh) [inline components/homepage/About/About.jsx]: remove the commented code

**Review by varun-singhh — CHANGES_REQUESTED** [review]

**Comment by VC6378** [comment]
I have removed the commented code, please check it @varun-singhh

**Review by varun-singhh — APPROVED** [review]

**Pull request merged**